tools/pot/docs/AccuracyAwareQuantizationUsage.md

```diff
@@ -11,27 +11,29 @@
 @endsphinxdirective
 
 ## Introduction
-This document assumes that you already tried [Default Quantization](@ref pot_default_quantization_usage) for the same model. In case when it introduces a significant accuracy degradation, the Accuracy-aware Quantization algorithm can be used to remain accuracy within the pre-defined range. This may cause a 
+For this article it'll be assumed that you already tried [Default Quantization](@ref pot_default_quantization_usage) for the same model. In case when it introduces a significant accuracy degradation, the Accuracy-aware Quantization algorithm can be used to remain accuracy within the pre-defined range. This may cause a 
```

Comment:
```suggestion
This article is based on the presumption that you have already tried [Default Quantization](@ref pot_default_quantization_usage) for the same model. When it introduces a significant accuracy degradation, the Accuracy-aware Quantization algorithm can be used to maintain accuracy within the pre-defined range. This may cause a 
```